components/Hero.vue

```diff
@@ -17,6 +17,7 @@
     </p>
     <img :src="hackCampLogo" class="hero-lhd-logo">
     <p class="hero-presents-text">
+      Powered by <a href="teck.com">Teck Resources</a>. <br>
```

Comment:
Should add https:// as a prefix for it to work.
```suggestion
      Powered by <a href="https://teck.com">Teck Resources</a>. <br>
```